sdcm/nemesis.py

```diff
@@ -747,7 +747,7 @@ def disrupt_start_stop_cleanup_compaction(self):
         of the command in (4).
         """
         compaction_args = self._prepare_start_stop_compaction()
-        trigger_func = partial(compaction_args.compaction_ops.trigger_cleanup_compaction)
+        trigger_func = partial(compaction_args.compaction_ops.trigger_cleanup_compaction, timeout=500)
```

Comment:
`500s` is not `10m`.
`600s` is `10m`.

So, need either update the commit description or change the timeout value to be `600` seconds.